aws-globalaccelerator-accelerator/src/main/kotlin/software/amazon/globalaccelerator/accelerator/ListHandler.kt

```diff
@@ -0,0 +1,47 @@
+package software.amazon.globalaccelerator.accelerator
+
+import com.amazonaws.services.globalaccelerator.model.Accelerator
+import com.amazonaws.services.globalaccelerator.model.ListAcceleratorsRequest
+import com.amazonaws.services.globalaccelerator.model.ListAcceleratorsResult
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy
+import software.amazon.cloudformation.proxy.HandlerErrorCode
+import software.amazon.cloudformation.proxy.Logger
+import software.amazon.cloudformation.proxy.OperationStatus
+import software.amazon.cloudformation.proxy.ProgressEvent
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest
+import software.amazon.globalaccelerator.accelerator.AcceleratorClientBuilder.client
+
+/**
+ * List handler implementation for accelerator resource.
+ */
+class ListHandler : BaseHandler<CallbackContext?>() {
+    override fun handleRequest(proxy: AmazonWebServicesClientProxy,
+                               request: ResourceHandlerRequest<ResourceModel>,
+                               callbackContext: CallbackContext?,
+                               logger: Logger): ProgressEvent<ResourceModel, CallbackContext?> {
+        logger.debug("List Accelerators Request: $request")
+        val agaClient = client
+        val listAcceleratorsRequest = ListAcceleratorsRequest().withNextToken(request.nextToken)
+        val listAcceleratorsResult = proxy.injectCredentialsAndInvoke(listAcceleratorsRequest, agaClient::listAccelerators)
+        val acceleratorList = convertAccleratorList(listAcceleratorsResult.accelerators)
+        return ProgressEvent.builder<ResourceModel, CallbackContext>()
+        	.status(OperationStatus.SUCCESS)
+		.resourceModels(acceleratorList)
```

Comment:
weird indenting here, can you please fix.

Reply:
Fixed